src/components/Sticker/index.tsx

```diff
@@ -68,7 +89,30 @@ export default function Sticker({
             : null}
         </span>
       )}
-      {children}
+      {showTooltip ? (
+        <Popover
+          position="top"
+          withArrow
+          opened={showPopover}
+          offset={{ mainAxis: 16 }}
+          styles={popoverStyles}
+        >
+          <Popover.Target>
+            <span
+              className={styles.text}
+              onMouseEnter={open}
+              onMouseLeave={close}
+            >
```

Comment:
did the `Tooltip` component from mantine not work here?

Reply:
No :(